cli/tests/pcluster/configure/test_pcluster_configure/test_vpc_automation_no_vpc_in_region_public/pcluster.config.ini

```diff
@@ -1,29 +1,33 @@
 [aws]
 aws_region_name = eu-west-1
 
+[global]
+cluster_template = default
+update_check = true
+sanity_check = true
+
+[aliases]
+ssh = ssh {CFN_USER}@{MASTER_IP} {ARGS}
+
 [cluster default]
 key_name = key1
-vpc_settings = default
+base_os = centos6
```

Comment:
minor: I'd use centos7 or alinux2 given that centos6 is going to be deprecated soon.

Reply:
I just reformatted the existing config files to HIT style.